pensieve/statistics.py

```diff
@@ -0,0 +1,160 @@
+import attr
+import cattr
+import mozanalysis.bayesian_stats.bayesian_bootstrap as mabsbb
+from google.cloud import bigquery
+from google.api_core.exceptions import NotFound
+from typing import Callable, Any, Dict, List, Tuple, Optional
+from pandas import DataFrame
+import pandas
+import numpy as np
+
+from pensieve.pre_treatment import PreTreatment, RemoveNulls
+
+
+@attr.s(auto_attribs=True)
+class StatisticResult:
+    """
+    Represents the resulting data after applying a statistic transformation
+    to metric data.
+    """
+
+    metric: str
+    statistic: str
+    parameter: float
```

Comment:
I'm not exactly sure what parameter should represent. It looks like most statistic function have multiple parameters (which could be represented in BigQuery as a map if needed).

Reply:
Yeah, it's intended to be statistic-dependent. What do you think about making this a Decimal type? Is that too precious?